src/doc/oiiotool.rst

```diff
@@ -2193,6 +2193,16 @@ current top image.
     image comprised of the subimages of all original images appended
     together.
 
+.. option:: --layersplit
+
+    Remove the top image from the stack, split it into its consituent
+    channel-name-based layers, and push them all onto the stack
+    (first to last).
+
+    By "layer" we mean a subset of the initial channels which, when named
+    using the convention "LAYERNAME.channelname", all share the same layer
+    name.
+
```

Comment:
Do you want to clarify what should happen to the channels that don't have any '.' at all? Are they gathered into one image together? Is it guaranteed to be the first image?

Reply:
Yes and yes ! (I've just checked the exr input code just to make sure)
I've updated the docs :)